site/src/pages/get-involved.mdx

```diff
@@ -14,9 +14,18 @@ The best place to start is by participating in one or more of our communication
 
 - [Discord](https://discord.gg/DEWjhSw): Ask a question, learn about what's happening, be alerted of upcoming meetings and opportunities to get involved in the Open UI community.
 - [Open UI GitHub](https://github.com/openui/open-ui): Our repo where we track issues, progress on spec proposals, and publish our research for public consumption
-- [Open UI Telecon Agendas](https://github.com/openui/open-ui/tree/main/meetings/telecon): We have weekly telecons at 11:00 AM Pacific Standard Time, you can find previous minutes and upcoming agendas here. If you want to get added to the calendar invite contact [Greg Whitworth](https://twitter.com/gregwhitworth) or [Una Kravets](https://twitter.com/Una) on [Discord](https://discord.com/invite/DEWjhSw) or [Twitter](https://twitter.com/openuicg?lang=en)
 - [Open UI community group](https://www.w3.org/community/open-ui/): In order to add substantial contributions to Open UI you need to join the Open UI CG
 
+## Joining the Open UI Telecon
+
+We have a weekly telecon on Thursdays at 11AM - 12PM PST. We post the agenda will be posted by Tuesday EOD on the Discord telecon channel along with the Jitsi video telecon link. The agenda and minute archives can be found [here](https://github.com/openui/open-ui/tree/main/meetings/telecon).
```

Comment:
```suggestion
We have a weekly telecon on Thursdays at 11AM - 12PM [Pacific Time](https://www.timeanddate.com/worldclock/usa/los-angeles). We post the agenda by end of day (Pacific time) Tuesday on the Discord telecon channel along with the Jitsi video telecon link. See the [agenda and minutes archives](https://github.com/openui/open-ui/tree/main/meetings/telecon).
```

Reply:
Rationale (i.e., what I wrote before turning it into an edit suggestion):

Maybe say "Pacific Time" rather than "PST" since it's DST for more than half of the year.  Perhaps link it to something like https://www.timeanddate.com/worldclock/usa/los-angeles as well?

Also maybe rephrase "Tuesday EOD" in a way that's a little clearer to people in other timezones or those who might not be familiar with "EOD"?